ts/boot/configureStoreAndPersistor.ts

```diff
@@ -283,9 +283,18 @@ const logger = createLogger({
 });
 
 // configure Reactotron if the app is running in dev mode
-export const RTron = isDevEnv ? configureReactotron() : {};
+const RTron = isDevEnv ? configureReactotron() : undefined;
```

Comment:
I think it would be a good idea to still export the `RTron` instance in order to access to other features.

Reply:
sure done in https://github.com/pagopa/io-app/pull/2056/commits/18b8d27df6ff6ae8a17f81ca4de372a05826604e